src/mainwindow.cpp

```diff
@@ -3751,3 +3739,25 @@ void MainWindow::setMargins(QMargins margins)
     ui->centralWidget->layout()->setContentsMargins(margins);
     m_trafficLightLayout.setGeometry(QRect(4 + margins.left(), 4 + margins.top(), 56, 16));
 }
+
+bool MainWindow::isTitleBar(const int &x, const int &y)
```

Comment:
Just two things:

* Please don't pass an `int` (4-byte value) as `const int &` (8-byte reference to that value). It's just more typing and makes things needlessly more expensive. :-)
* This function should be `const`.

Reply:
>Please don't pass an `int` (4-byte value) as `const int &` (8-byte reference to that value)

TIL... 😅 

>This function should be `const`.

Done. Thanks!